packages/purpose-process/src/services/purposeService.ts

```diff
@@ -1100,6 +1320,18 @@ const replacePurposeVersion = (
   };
 };
 
+const archiveActiveAndSuspendedPurposeVersions = (
+  versions: PurposeVersion[]
+): PurposeVersion[] =>
+  versions.map((v) =>
+    match(v.state)
+      .with(purposeVersionState.active, purposeVersionState.suspended, () => ({
+        ...v,
+        state: purposeVersionState.archived,
```

Comment:
Do `updatedAt` need to be updated as well?

Reply:
Good point, didn't notice any updating of the `updatedAt` in the scala code. I would ask to @galales

Reply:
Fixed!